src/scripts/enable_unattended_upgrades.sh

```diff
@@ -97,6 +99,28 @@ write_content_to_file() {
     fi
 }
 
+# Make sure needrestart is installed and runs automatically to avoid user intervention
+
+# Check if needrestart is installed
+if command -v needrestart &> /dev/null; then
+    # Update package repositories and the needrestart package
+    apt-get update
+    apt-get install --only-upgrade -y needrestart
```

Comment:
By default `apt-get install -y needrestart` will upgrade the package if it is installed and install it if not, so this check is not needed.

Reply:
Fixed in https://github.com/dappnode/DNP_CORE/pull/142/commits/cb1ec3ff884c302a5ceebd45e0b7826a7197ca14